site/docs/v1/tech/troubleshooting.adoc

```diff
@@ -113,12 +114,24 @@ When the above modules are available to your Java runtime, configure JMX by pass
 
 include::docs/v1/tech/email-templates/_email-troubleshooting.adoc[]
 
-== Troubleshooting SAML or OIDC Identity Provider Configuration
+== Troubleshooting Identity Provider Configuration
```

Comment:
Apple will also require CORS configuration - they `POST` back to the `/oauth2/callback` as well. 

Currently this means we will require an allowed origin of  `https://appleid.apple.com` - and an allowed HTTP `POST `method. 

https://fusionauth.io/docs/v1/tech/identity-providers/apple/#cors-configuration